apps/mobile/src/components/widgets/account/ledger-badge.tsx

```diff
@@ -0,0 +1,23 @@
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import { Box, Text, Theme } from '@leather.io/ui/native';
+
+export function LedgerBadge() {
+  const theme = useTheme<Theme>();
+  return (
+    <Box
+      position="absolute"
+      p="1"
+      borderColor="ink.border-transparent"
+      borderRadius="xs"
+      borderWidth={1}
+      right={theme.spacing[5]}
+      top={theme.spacing[5]}
```

Comment:
can't this just be {5}

Reply:
I will experiment with this, I copied this from elsewhere. 

If we set it to `top={5}` I think that's 5 pixel units but I see other areas where we set `px='5'` and that uses our tokens so I will try and refactor away `theme.spacing[5]` if possible. 

I opened a task for it so we don't forget https://github.com/leather-io/issues/issues/321